week 1/34th-Sopt-Seminar/ViewController/LoginViewController.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  LoginViewController.swift
+//  34th-Sopt-Seminar
+//
+//  Created by 오연서 on 3/30/24.
+//
+
+import Foundation
+import UIKit
+
+class LoginViewController: UIViewController {
+    
+    let pretendardSB = NSMutableAttributedString(string: "Pretendard-SemiBold")
+
+    private let titleLabel: UILabel = {
+        let label = UILabel(frame: CGRect(x: 69, y: 161, width: 236, height: 44))
+        label.text = "동네라서 가능한 모든것\n당근에서 가까운 이웃과 함께해요."
+        label.font = UIFont(name: "Pretendard-SemiBold", size: 18)
+        label.textColor = .black
+        label.numberOfLines = 2
+        label.textAlignment = .center
+        return label
+    }()
+    
+    private let idTextField: UITextField = {
+        let textField = UITextField(frame: CGRect(x: 20, y: 276, width: 335, height: 52))
+        textField.backgroundColor = UIColor(hexCode: "DDDEE3")
+        textField.leftView = UIView(frame: CGRect(x: 0, y: 0, width: 23, height: 52))
+        textField.leftViewMode = .always
+        textField.attributedPlaceholder = NSAttributedString(
+            string: "아이디",
+            attributes: [NSAttributedString.Key.foregroundColor: UIColor(hexCode: "ACB0B9")]
+        )
+        textField.layer.cornerRadius = 5
+        return textField
+    }()
+    
+    private let passwordTextField: UITextField = {
+        let textField = UITextField(frame: CGRect(x: 20, y: 335, width: 335, height: 52))
+        textField.backgroundColor = UIColor(hexCode: "DDDEE3")
```

Comment:
Assets에 Color set으로 색깔을 넣어두면 (ex) subRed 이름으로 색을 넣어두면)
```suggestion
        textField.backgroundColor = .subRed
```
이런식으로도 쓸수 있는 것으로 알고 있습니다!